src/main/java/com/aws/greengrass/mqttclient/v5/Subscribe.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.mqttclient.v5;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+@Builder
+@Value
+public class Subscribe {
+    @NonNull String topic;
+    @Builder.Default
+    @NonNull QOS qos = QOS.AT_LEAST_ONCE;
+
+    Boolean noLocal;
+    Boolean retainAsPublished;
+    RetainHandlingType retainHandlingType;
```

Comment:
Explicit defaults?

Reply:
defaults are null which is defined in the spec. Want me to set them null here?

Reply:
updated.